tests/test_minimal/test_converter.py

```diff
@@ -49,17 +52,17 @@ class TestNWBConverterAndPipeInitialization(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         class InterfaceA(BaseDataInterface):
-            def __init__(self, **source_data):
-                super().__init__(**source_data)
+            def __init__(self):
```

Comment:
One change, that is probably good to have to added required specificity - any child of the `BaseDataInterface` must explicitly define all keyword arguments and attach annotation typing to them (or else override the new class method for getting the pydantic model for the source using `exclude` on the thing you want to skip validation of)